geotrek/outdoor/locale/it/LC_MESSAGES/django.po

```diff
@@ -292,6 +295,9 @@ msgstr ""
 msgid "Courses"
 msgstr ""
 
+msgid "Parcours"
```

Comment:
No need for these if `_("Parcours")` is properly replaced with `_("Course")` everywhere